### DISCUSSION TITLE
Make errors installing 8.9.2 on Ubuntu 20.04 - libx265.so.165 & libIlmImf-2_2.so.23

### DISCUSSION THREAD

**Discussion by jumpinvestor** [libvips/libvips#2008] [Q&A]
Hi, I've been trying to install 8.9.2 on Ubuntu 20.04. Each of the methods in the instructions has these errors on make:

libx265.so.165, needed by /usr/local/lib/libheif.so, not found
libIlmImf-2_2.so.23, needed by /usr/local/lib/libMagickCore-7.Q16HDRI.so, not found

<img width="977" alt="Screen Shot 2020-06-10 at 5 46 51 PM" src="https://user-images.githubusercontent.com/14829156/84414540-c5492180-abc6-11ea-81e3-f4627573d8d8.png">

This is an installation to use with the Sharp Node package (see [#2253](https://github.com/lovell/sharp/issues/2253)).

Sharp returns a similar error:

<img width="779" alt="Screen Shot 2020-06-10 at 5 47 06 PM" src="https://user-images.githubusercontent.com/14829156/84415043-0d684400-abc7-11ea-9097-7bce9ec86790.png">

The part that I am having difficulty with is that I've verified that I have libx265.so.179 installed, but they're both requiring 165. 

Wiping everything and reinstalling fresh has the same result.

I can get everything to work with a previously installed version, 8.8.3, just not with 8.9.2.

**Comment by jcupitt** [comment]
Hello, I think this is a bug in imagemagick. You need to configure imagemagick with `--with-modules`.

**Comment by jcupitt** [comment]
Oh, or build libvips using `--without-magick`, of course.
